### PR TITLE
Added get_temperature()

### PR DESCRIPTION
**Pull request by Lildirt**
Fetch the current temperature of the location given.

### CODE REVIEW

File: src/main/java/com/laytonsmith/core/functions/World.java [hunk 1]
```diff
@@ -1801,4 +1801,54 @@ public Version since() {
...
+
+		@Override
+		public Construct exec(Target target, Environment environment, Construct... args) throws ConfigRuntimeException {
+			MCPlayer player = environment.getEnv(CommandHelperEnvironment.class).GetPlayer();
+			MCWorld world = null;
+			if(player != null) { world = player.getWorld(); }
+			MCLocation loc = ObjectGenerator.GetGenerator().location(args[0], world, target);
+			double temp;
+			temp = loc.getBlock().getTemperature();
+			return new CDouble(temp, target);
```
Comment (jb-aero) [inline src/main/java/com/laytonsmith/core/functions/World.java]: In this case, you can combine that last 3 lines into "return new CDouble(loc.getBlock().getTemperature(), target);

File: src/main/java/com/laytonsmith/core/functions/World.java [hunk 2]
```diff
@@ -1801,4 +1801,54 @@ public Version since() {
...
+		@Override
+		public Boolean runAsync() {
+			return false;
+		}
+
+		@Override
+		public Construct exec(Target target, Environment environment, Construct... args) throws ConfigRuntimeException {
+			MCPlayer player = environment.getEnv(CommandHelperEnvironment.class).GetPlayer();
+			MCWorld world = null;
+			if(player != null) { world = player.getWorld(); }
```
Comment (jb-aero) [inline src/main/java/com/laytonsmith/core/functions/World.java]: She may ask you to follow the formatting guidelines and make this line 3 lines.
    if(player != null) { 
        world = player.getWorld();
    }

**Comment by Lildirt** [comment]
Amended those issues you brought up. Seems like IDEA wanted to make the if contents a one-liner for some reason.

**Comment by jb-aero** [comment]
Yeah, it does that based on your settings. It IS a valid format, but for consistency and quick reading, LC likes the content on a separate line from the condition.

**Comment by LadyCailin** [comment]
Yes, please put ifs on multiple lines, barring some exceptional circumstance (like if you have a huge chain of if/elses that are one liners, and a switch statement won't work for whatever reason).

**Pull request merged**